### PR TITLE
fix debt/all should return debt for a particular user only

### PR DESCRIPTION
**Pull request by judgegodwins**
# Description
Fix. /debt all should returnall debts for only a user and not all debts for all users in DB
Fix for #164 

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:
- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

Screenshot from 2020-06-28 13-55-35

Screenshot from 2020-06-28 13-56-11

Now /debt/all returns all debts according to a users `phone_number`. Notice the `phone_number` property.

I also changed `phone_number` to be a String in the user schema because if it's a number it'll cause a lot of problems

### CODE REVIEW

File: models/debt_reminders.js [hunk 1]
```diff
@@ -1,6 +1,11 @@
 const mongoose = require('mongoose')
 
 const DebtReminder = new mongoose.Schema({
+  phone_number: {
+    type: String,
```
Comment (nerdyphil) [inline models/debt_reminders.js:5]: Change to number

File: models/user.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ const mongoose = require('mongoose')
 
 
 const User = new mongoose.Schema({
-  phone_number: { type: Number, required: true,  unique: true }, 
+  phone_number: { type: String, required: true,  unique: true },
```
Comment (nerdyphil) [inline models/user.js:5]: No no no... It has to be a number

**Comment by judgegodwins** [comment]
working on it

**Comment by judgegodwins** [comment]
@nerdyphil I'm done
Screenshot from 2020-06-28 20-04-55

**Comment by chinazaaa** [comment]
Pull recent changes and do another PR

**Pull request closed without merging**